src/sinks/azure_blob/service.rs

```diff
@@ -0,0 +1,91 @@
+use bytes::Bytes;
```

Comment:
Seems like this file should be `request_builder`?, it doesn't contain a service?